templates/addEvent.phtml

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Add Event</title>
+        <link rel="stylesheet" type="text/css" href="css/genericStyles.css"/>
+        <link rel="stylesheet" href="css/addEventPage.css">
+    </head>
+    <body>
+        <main class="addEventPageWrapper">
+            <h2 id="heading">Add Event</h2>
+            <form action="/addEvent" method="POST" id="addEventForm">
+                <div class="eventDataRow">
+                    <label for="eventName" class="eventLabel">Event Name: </label>
+                    <input type="text" name="eventName" id="eventName" class="eventInput">
+                </div>
+                <div class="eventDataRow">
+                    <label for="location" class="eventLabel">Location: </label>
+                    <input type="text" name="location" id="location" class="eventInput">
+                </div>
+                <div class="eventDataRow">
+                    <label for="eventType" class="eventLabel">Type of Event: </label>
+                    <input type="text" name="eventType" id="eventType" class="eventInput">
+                </div>
+                <div class="eventDataRow">
+                    <label for="date" class="eventLabel">Date: </label>
+                    <input type="date" name="date" id="date" class="eventInput">
```

Comment:
type date has poor support.  Ensure using date meets PO's requirements as I believe we spoke about using a js plugin instead of type date.

Reply:
Solved in 74505f7

Reply:
Okay but please see comment on line 57